packages/yarnpkg-pnpify/sources/generateSdk.ts

```diff
@@ -62,10 +62,32 @@ export const generateEslintWrapper = async (projectRoot: PortablePath, target: P
   await addVSCodeWorkspaceSettings(projectRoot, {'eslint.nodePath': NodeFS.fromPortablePath(ppath.relative(projectRoot, ppath.dirname(eslint)))});
 };
 
+const isPackageInstalled = (name: string): boolean => {
+  try {
+    dynamicRequire(name);
```

Comment:
We can just check whether it exists or not:
```suggestion
    dynamicRequire.resolve(name);
```